db/seeds.rb

```diff
@@ -1,15 +1,30 @@
 require './app/models/merchant.rb'
+require './app/models/invoice.rb'
+require './app/models/item.rb'
 require 'csv'
 require 'pry'
 
```

Comment:
you could also call the DatabaseCleaner gem in here (from our film file spec_helper) to empty your database before seeding